packages/template-retail-react-app/app/pages/product-detail/index.jsx

```diff
@@ -155,105 +147,36 @@ const ProductDetail = ({category, product, isLoading}) => {
-                                    id: 'product_detail.accordion.message.coming_soon'
-                                })}
-                            </AccordionPanel>
-                        </AccordionItem>
-
-                        {/* Reviews */}
-                        <AccordionItem>
-                            <h2>
-                                <AccordionButton height="64px">
-                                    <Box flex="1" textAlign="left" fontWeight="bold" fontSize="lg">
-                                        {formatMessage({
-                                            defaultMessage: 'Reviews',
-                                            id: 'product_detail.accordion.button.reviews'
-                                        })}
-                                    </Box>
-                                    <AccordionIcon />
-                                </AccordionButton>
-                            </h2>
-                            <AccordionPanel mb={6} mt={4}>
-                                {formatMessage({
-                                    defaultMessage: 'Coming Soon',
-                                    id: 'product_detail.accordion.message.coming_soon'
-                                })}
-                            </AccordionPanel>
-                        </AccordionItem>
-
-                        {/* Questions */}
-                        <AccordionItem>
-                            <h2>
-                                <AccordionButton height="64px">
-                                    <Box flex="1" textAlign="left" fontWeight="bold" fontSize="lg">
-                                        {formatMessage({
-                                            defaultMessage: 'Questions',
-                                            id: 'product_detail.accordion.button.questions'
-                                        })}
-                                    </Box>
-                                    <AccordionIcon />
-                                </AccordionButton>
-                            </h2>
-                            <AccordionPanel mb={6} mt={4}>
-                                {formatMessage({
-                                    defaultMessage: 'Coming Soon',
-                                    id: 'product_detail.accordion.message.coming_soon'
-                                })}
-                            </AccordionPanel>
-                        </AccordionItem>
-                    </Accordion>
-                    <Box display={['none', 'none', 'none', 'block']} flex={4}></Box>
-                </Stack>
+                {product?.type.set ? (
+                    // Product Set: render the child products
+                    product.setProducts.map((childProduct) => (
+                        <Fragment key={childProduct.id}>
+                            <ProductView
+                                product={childProduct}
+                                isSetProduct={true}
```

Comment:
I added this `isSetProduct` for the ProductView component (and also the related React hooks) because given solely the data for a _child_ product, it is not possible to identify whether the product belongs to a set or not.

Reply:
Have you thought about other product types that we currently don't support that we might in the future. And how using `isSetProduct` as a prop will age? 

Side note: Is `isSetProduct` short for `isProductSetProduct`

Reply:
@bendvc Good questions.. I don't know yet at this time, but there's a separate refactoring ticket for supporting other product types.🤔 However, I'll think about it a bit more at this stage anyways.

Reply:
`isSetProduct` comes from the way SCAPI calls the child products. The data for a product set has `setProducts: [...]`.